src/components/PeopleTable/SegmentAssignmentModal.jsx

```diff
@@ -0,0 +1,246 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  chakra,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'chakra' is defined but never used.